### PR TITLE
INTDEV-368 Validate template cards

### PR DESCRIPTION
**Pull request by Fuzzbender**
Add card content validation to template cards. 
They have similar requirements as project cards, but workflowState must be empty.

### CODE REVIEW

File: tools/data-handler/test/test-data/valid/minimal/.cards/local/templates/test-template/template.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
-  "buttonLabel": "Create a New Decision",
-  "namePrompt": "The title of the new decision:"
+    "buttonLabel": "",
```
Comment (samutoljamo-pareto) [inline tools/data-handler/test/test-data/valid/minimal/.cards/local/templates/test-template/template.json:2]: We just removed buttonLabel/namePrompt, so either this must be reverted and rebased or just remove them also from here

File: tools/data-handler/src/validate.ts [hunk 2]
```diff
@@ -423,35 +428,54 @@ export class Validate {
...
         }
       } else {
-        validationErrors += `Workflow of '${cardType.workflow}' cardtype '${cardType?.name}' does not exist in the project`;
+        validationErrors.push(
+          `Workflow of '${cardType.workflow}' cardtype '${cardType?.name}' does not exist in the project`,
+        );
       }
     }
-    return validationErrors;
+    return validationErrors.join('\n');
```
Comment (samutoljamo-pareto) [inline tools/data-handler/src/validate.ts:478]: I think it would be better to represent validation errors using a list of either objects or just strings. But that deserves an own PR

**Review by samutoljamo-pareto — APPROVED** [review]

File: tools/data-handler/test/test-data/valid/minimal/.cards/local/templates/test-template/template.json (hunk 1 above)
Comment (Fuzzbender) [reply tools/data-handler/test/test-data/valid/minimal/.cards/local/templates/test-template/template.json:2]: I'll rebase this and fix if there are issues.

File: tools/data-handler/src/validate.ts (hunk 2 above)
Comment (samutoljamo-pareto) [reply tools/data-handler/src/validate.ts:478]: Looks like #130 does that :smile:
